src/open_samus_returns_rando/lua_editor.py

```diff
@@ -212,6 +212,15 @@ def _create_custom_init(self, editor: PatcherEditor, configuration: dict) -> str
         configuration_identifier: str = configuration["configuration_identifier"]
         enable_remote_lua: bool = configuration.get("enable_remote_lua", False)
 
+        starting_scenario = starting_location["scenario"]
+        scenario_bmsld = editor.get_scenario(starting_scenario)
+        starting_actor = starting_location["actor"]
+        for actor in scenario_bmsld.raw.actors[5]:
+            if starting_location["actor"] == actor:
+                break
+        else:
```

Comment:
i'm not thanatos, but i would prefer to have a temp variable instead of using for...else.

Reply:
How would this help?

Reply:
more readable for me. I find for-else to be obscure syntax.

Reply:
I just updated it. Definitely cleaner.